pkg/workload/histogram/exporter/util.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package exporter
+
+import (
+	"regexp"
+	"time"
+
+	"github.com/codahale/hdrhistogram"
+	"github.com/gogo/protobuf/proto"
+	io_prometheus_client "github.com/prometheus/client_model/go"
+)
+
+func sanitizeOpenmetricsLabels(input string) string {
+	regex := regexp.MustCompile(`[^a-zA-Z0-9_]`)
+	sanitized := regex.ReplaceAllString(input, "_")
+	firstCharRegex := regexp.MustCompile(`^[^a-zA-Z_]`)
+	sanitized = firstCharRegex.ReplaceAllString(sanitized, "_")
+	return sanitized
+}
+
+// ConvertHdrHistogramToPrometheusMetricFamily converts a Hdr histogram into MetricFamily which can be used
```

Comment:
Would help understanding if you could add one simple example in comments for the desired input and output formats.